usr/share/rear/prep/Linux-s390/034_check_config.sh

```diff
@@ -0,0 +1,8 @@
+if [ "$OUTPUT" = "IPL" ]; then
+    LogPrint "WARNING: OUTPUT=IPL is deprecated. Please, use the equivalent OUTPUT=RAMDISK instead."
```

Comment:
LogPrint messages are only shown in verbose mode.
Perhaps LogPrintError is better here,
see the LogPrintError description in
lib/_input-output-functions.sh currently online at
https://github.com/rear/rear/blob/master/usr/share/rear/lib/_input-output-functions.sh#L527
and for a recent example see
https://github.com/rear/rear/blob/master/usr/share/rear/prep/PXE/default/010_PXE_check.sh

Reply:
And please don't use "please" in user messages.
Computers (i.e. machines) should not beg
the user (i.e. humans) to do something.
So better simple and straightforward:
```
OUTPUT=IPL is deprecated. Use OUTPUT=RAMDISK instead.
````
It is questionable if we should shout at the user with
```
WARNING: ...
```
Probably
```
Warning: ...
```
is better.
Perhaps best is to leave out the word "warning" at all, cf.
https://schlomo.schapiro.org/2015/04/warning-is-waste-of-my-time.html
I.e. reading the word "warning" could be perceived
as an annoyance by users.

Reply:
Addressed in 75ea83f469a312689e98d57203f3e5d5edb37fe1.  I've decided to go with `Warning:` because there is no guarantee for how long will be the `OUTPUT=IPL` fallback supported.